<launch>
    <node name="move_and_back" pkg="gen5" type="move_and_back.py" output="screen"/>
    <node name="sigint_catcher" pkg="gen5" type="sigint_catcher.py" output="screen"/>
</launch>